docs/ml-features.md

```diff
@@ -145,9 +148,11 @@ for more details on the API.
  passed to other algorithms like LDA.
 
  During the fitting process, `CountVectorizer` will select the top `vocabSize` words ordered by
- term frequency across the corpus. An optional parameter "minDF" also affects the fitting process
+ term frequency across the corpus. An optional parameter `minDF` also affects the fitting process
  by specifying the minimum number (or fraction if < 1.0) of documents a term must appear in to be
- included in the vocabulary.
+ included in the vocabulary. Another optional binary toggle parameter controls the output vector.
```

Comment:
The difference in results is a bit puzzling.  I'm getting the same thing as @MLnick.  Could you both look at the output of `df.stat.approxQuantile("hour", Array(1.0/3, 2.0/3), relativeError=0.001)`.  I get the following on the DataFrame above:

```
scala> df.stat.approxQuantile("hour", Array(1.0/3, 2.0/3), relativeError=0.001)
res8: Array[Double] = Array(2.2, 5.0)
```